package.json

```diff
@@ -17,6 +17,12 @@
     "Daniel Reguero"
   ],
   "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts",
```

Comment:
So TS types-files are from now on always bundled with the NPM package, right?

Reply:
Correct